### PR TITLE
feat(assistant): add paperclip icon

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-1185

2024-08-19-183530_480x729_scrot
2024-08-19-183524_1919x779_scrot

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview b1cc6d03-1b9e-4536-b5ee-dc186c55218c | [dev](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/b1cc6d03-1b9e-4536-b5ee-dc186c55218c/services/general) |
 /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Take rest sometimes

File: libs/shared/assistant/feature/src/lib/assistant-trigger/assistant-trigger.tsx [hunk 1]
```diff
@@ -9,20 +12,39 @@ export interface AssistantTriggerProps {
...
-        <Icon
-          iconStyle="light"
-          iconName="question"
-          className="text-2xl group-hover:animate-[shake_0.6s_cubic-bezier(0.36,0.07,0.19,0.97)_both]"
-        />
-        <span className="pointer-events-none absolute left-0 top-0 h-[150px] w-[150px] bg-gradient-to-b from-transparent from-0% via-neutral-50 via-50% to-transparent to-100% opacity-0 group-hover:animate-[shineAssistantTrigger_1s_linear_forwards] group-hover:opacity-20"></span>
-      </button>
+      {match(assistantIcon)
+        .with('QUESTION_MARK', () => (
+          <button
```
Comment (RemiBonnet) [inline libs/shared/assistant/feature/src/lib/assistant-trigger/assistant-trigger.tsx:21]: Could you create a component to avoid duplicate this DOM?

**Review by RemiBonnet — CHANGES_REQUESTED** [review]
Well done @ctjhoa, only one point the rest LGTM 👍

File: libs/shared/assistant/feature/src/lib/assistant-trigger/assistant-trigger.tsx (hunk 1 above)
Comment (ctjhoa) [reply libs/shared/assistant/feature/src/lib/assistant-trigger/assistant-trigger.tsx]: I've factorize the code without creating a new component
https://github.com/Qovery/console/pull/1571/commits/e2777853b02a92e167237fcc75e35c6f29760c0c

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**